automation/TaskManager.py

```diff
@@ -425,13 +423,13 @@ def _issue_command(self, browser: Browser,
                 command_status = 'timeout'
                 self.logger.info(
                     "BROWSER %i: Timeout while executing command, %s, killing "
-                    "browser manager" % (browser.crawl_id, command[0]))
+                    "browser manager" % (browser.crawl_id, repr(command)))
 
             self.sock.send(("crawl_history", {
                 "crawl_id": browser.crawl_id,
                 "visit_id": browser.curr_visit_id,
-                "command": command[0],
-                "arguments": str(command_arguments),
+                "command": repr(command),
```

Comment:
```suggestion
                "command": type(command),
```
Because otherwise you're writing the same string twice